src/elm/Pages/Secrets/Form.elm

```diff
@@ -219,6 +224,44 @@ viewImagesInput secret imageInput =
         ]
 
 
+{-| radio : takes current value, field id, title for label, and click action and renders an input radio.
+-}
+radio : String -> String -> Field -> msg -> Html msg
+radio value field title msg =
+    div [ class "form-control", Util.testAttribute <| "secret-radio-" ++ field ]
+        [ input
+            [ type_ "radio"
+            , id <| "radio-" ++ field
+            , checked (value == field)
+            , onClick msg
+            ]
+            []
+        , label [ class "form-label", for <| "radio-" ++ field ] [ strong [] [ text title ] ]
+        ]
+
+
+{-| allowCommandCheckbox : renders checkbox inputs for selecting allowcommand
+-}
+viewAllowCommandCheckbox : SecretForm -> Html Msg
+viewAllowCommandCheckbox secretUpdate =
+    section [ class "type", Util.testAttribute "" ]
+        [ h4 [ class "field-header" ]
+            [ text "Allow Commands"
+            , span [ class "field-description" ]
+                [ text "( "
+                , em [] [ text "\"No\" will disable this secret in " ]
+                , code [] [ text "commands" ]
+                , text " )"
+                ]
+            ]
```

Comment:
I think the `h4` is part of the reason for the header being nudged over in this case. Should be able to make it more consistent by modelling it after: https://github.com/go-vela/ui/blob/067939009c9c405ff333a4d9879418d612ce8cca/src/elm/Pages/Secrets/Form.elm#L147-L155